### PR TITLE
Regenerate computer vision SDK

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: eng/versioning/version_data.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ com.microsoft.azure.cognitiveservices:azure-cognitiveservices-newssearch;1.1.0-b
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-videosearch;1.1.0-beta.1;1.1.0-beta.1
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-visualsearch;1.1.0-beta.1;1.1.0-beta.1
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-websearch;1.1.0-beta.1;1.1.0-beta.1
-com.microsoft.azure.cognitiveservices:azure-cognitiveservices-computervision;1.0.6-beta;1.0.7-beta
+com.microsoft.azure.cognitiveservices:azure-cognitiveservices-computervision;1.0.7-beta;1.0.8-beta
```
Comment (alzimmermsft) [inline eng/versioning/version_data.txt:21]: Are there plans on ever having this following semver we use everywhere else?

**Review by alzimmermsft — APPROVED** [review]

File: eng/versioning/version_data.txt (hunk 1 above)
Comment (jianghaolu) [reply eng/versioning/version_data.txt:21]: The track 1 cognitive services libraries have been released in this version format - they will follow the track 2 versioning in track 2 libs.

Comment (srnagar) [reply eng/versioning/version_data.txt:21]: Since this release has new APIs added, the version should be `1.1.0-beta`

**Review by srnagar — APPROVED** [review]

File: eng/versioning/version_data.txt (hunk 1 above)
Comment (jianghaolu) [reply eng/versioning/version_data.txt:21]: Synced offline - will keep track 1 legacy versioning

**Pull request merged**